### PR TITLE
Release 0.1+backports

### PR DESCRIPTION
**Pull request by frezbo**

### CODE REVIEW

File: docs/resources/machine_configuration_controlplane.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ resource "talos_machine_configuration_controlplane" "machineconfig_cp" {
 ### Optional
 
 - `config_patches` (List of String) config patches to apply to the generated config
-- `config_version` (String) the desired machine config version to generate
+- `config_version` (String) the desired machine config version to generate (the `version` field of the generated config, currently `v1alpha1`)
 - `docs_enabled` (Boolean) whether to render all machine configs adding the documentation for each field
 - `examples_enabled` (Boolean) whether to render all machine configs with the commented examples
 - `kubernetes_version` (String) desired kubernetes version to run
-- `talos_version` (String) The version of Talos for which to generate configs
+- `talos_version` (String) The version of Talos for which to generate configs. **Note**: This parameter defines the machine config schema version generated. To override the installer image (and actually install a different version of talos than the default) please use a config patch.
```
Comment (TimJones) [inline docs/resources/machine_configuration_controlplane.md:45]: Reading the note: is there a way to change this to be `schema_version` or something more obvious?

Comment (frezbo) [reply docs/resources/machine_configuration_controlplane.md:45]: This was just from the cli:

` --talos-version string                     the desired Talos version to generate config for (backwards compatibility, e.g. v0.8)`

maybe we should first fix with a better name, then update here.

Comment (TimJones) [reply docs/resources/machine_configuration_controlplane.md:45]: Aaaahhh, OK.

```suggestion
- `talos_version` (String) The version of Talos for which to generate configs. **Note**: This parameter defines the desired Talos schema version of configuration generated for backwards compatibility. To change the version of Talos installed, please use a config patch.
```

Comment (frezbo) [reply docs/resources/machine_configuration_controlplane.md:45]: so basically this allows to generate talos machine config for older talos versions

**Pull request merged**